packages/terra-data-grid/src/subcomponents/ColumnHeader.jsx

```diff
@@ -14,31 +14,48 @@ const propTypes = {
    * String that specifies the column header height. Any valid CSS height value accepted.
    */
   headerHeight: PropTypes.string.isRequired,
-
   /**
-   * Number that specifies the height of the data grid in pixels.
-   */
+  * Number that specifies the height of the data grid in pixels.
+  */
   tableHeight: PropTypes.number,
-
+  /**
+   * Column index for cell that can receive tab focus.
+   */
+  activeColumnIndex: PropTypes.number,
+  /**
+   * Specifies if resize handle should be active.
+   */
+  activeColumnResizing: PropTypes.bool,
```

Comment:
nit: If this is a bool, it feels weird that we don't prefix this as `is`. Non-blocking though.

Reply:
Updated variable name here: 3abc11520c3c9130410fc26d759cea04bf1d41bf